Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM docker.pkg.github.com/caritasdeutschland/caritas-onlineberatung-nginx/nginx-image:dockerimage.v.1
```

Comment:
I'm using the Caritas image here currently. If there will be a separate one for the Diakonie we can use that one later. Or we just keep using the one from Caritas.